BlazarUI/app/scripts/pages/app.jsx

```diff
@@ -4,12 +4,15 @@ import BuildsSidebarContainer from '../components/sidebar/BuildsSidebarContainer
 import Modal from 'react-bootstrap/lib/Modal';
 import Button from 'react-bootstrap/lib/Button';
 import Input from 'react-bootstrap/lib/Input';
+import {bindAll} from 'underscore';
 
 class App extends Component {
 
   constructor() {
+    bindAll(this, 'collapseSidebar');
     this.state = {
-      showModal: (!window.config.apiRoot)
+      showModal: (!window.config.apiRoot),
+      sidebarCollapsed: localStorage.sidebarCollapsed === 'true' ? true : false
```

Comment:
You don't need the ternary operator, it can be rewritten as:

```javascript
  sidebarCollapsed: localStorage.sidebarCollpased === 'true'
```

Also, @benrlodge there are a couple of concerning bits in this module:

1. It's accessing and modifying globals directly (`localStorage` and `window.config.apiRoot`). These globals are also used to determine the UI state. A react component ideally should be a pure function of `state`, `props`, and `context`.
2. For some reason it's modifying `this.state` direct then calling `this.forceUpdate()` instead of just using `this.setState`

Suggested solutions:

1. Treat reading from globals the same way you would reading from an API. I'm not familiar with the Flux framework you're using but if you have a model for an API then you should have a model for `localStorage` that this module can subscribe to. This will also make your components easier to unit test since you won't have to stub `localStorage` and `window` reads.
2. Use `this.setState`